exporter/loadbalancingexporter/resolver_k8s.go

```diff
@@ -197,7 +211,8 @@ func (r *k8sResolver) resolve(ctx context.Context) ([]string, error) {
 	r.updateLock.Lock()
 	r.endpoints = backends
 	r.updateLock.Unlock()
-	_ = stats.RecordWithTags(ctx, k8sResolverSuccessTrueMutators, mNumBackends.M(int64(len(backends))))
+	r.telemetry.LoadbalancerNumBackends.Record(ctx, int64(len(backends)), metric.WithAttributeSet(k8sResolverAttrSet))
```

Comment:
It looks like this is only recorded when the list of backends changes, but the metric description is simply "current number". I would have assumed from the description that this would simply be recorded at some given frequency, not necessarily related to the value changing. I don't have much experience here, but would we be spamming to record every time here? Or maybe it's agreed upon that it's only useful to log when the value changes?

Reply:
I believe this wouldn't be causing too much noise, but given that this is a gauge, I would give a new value only when a new value is observed.